docs/self-hosted-getting-started.md

```diff
@@ -0,0 +1,99 @@
+# Quick Start: Run a self-hosted database instance with automatic vectorization
+
+If you want to see how it works, follow these steps to setup 
+a self-hosted database instance with automatic embedding vectorization:
+
+## Pre-requisites
+- Docker installed on your machine
+- An OpenAI API key. You can create one [here](https://platform.openai.com/api-keys).
+
+## Setup
+
+1. Run the following docker-compose file to start a TimescaleDB instance and the Vectorizer worker: 
```

Comment:
We should add the command that starts the docker-compose. 

We can also start just the db with `docker compose up -d db`, let the user run through al the setup. Then tell them to run `docker compose up -d pgai-vectorizer-worker` to start the vectorizer worker. 

If they start everything together, the pgai-vectorizer-worker container will fail because the pgai is not installed. 

Another option is to do setup in a sql init file, and have that be loaded when the service starts.

Reply:
I like the separate compose up commands I think. Keeps things simple, and I can remove the `depends_on` then as well.

I want to really get to that successful search-query in the least complex way possible.

Reply:
You can leave the depends_on , that way if you stop and restart everything, then the worker will wait for the DB to be ready to accept connections. Users will be fine, since data will be stored in the volume, so they don't have to go through all the set up again.

Reply:
Not sure about this, I prefer to keep it as simple as possible. The more i add to the compose the more complex it makes it look to a potential user. And we want the setup to look and be as trivial as possible imo.

I could make the same argument about removing the volume actually. Maybe somewhere we have to draw a line between usability and ease of understanding.